src/lib/helpers/date.ts

```diff
@@ -128,6 +128,11 @@ export const localeTimezoneName = () => {
     return dateWithTimezone.split(', ')[1];
 };
 
+export const localeShortTimezoneName = () => {
+    const timezone = localeTimezoneName();
+    return timezone.match(/[A-Z]/g)?.join('') || timezone;
+};
+
```

Comment:
Why can't we use `shortGeneric` for `timeZoneName` in `Intl.DateTimeFormatOptions` instead?

Reply:
1. doesn't fit

  ![image](https://github.com/user-attachments/assets/4b6f80db-2a09-4bfc-869d-a575b5a3cdd8)

2. using regex replace gives a better tz name like `IST` whereas using `shortGeneric` gives `India Time`

  ![image](https://github.com/user-attachments/assets/e5a5b9f1-0963-42ed-b4c4-02cac3546e1c)

Reply:
so frustrating...it looks like we should be using `short`, but it'll show as `GMT+5:30` rather than `IST` because `IST` is vague and can be Israel Standard Time or Irish Standard Time, etc.

Reference: https://github.com/tc39/proposal-temporal/issues/2257#issuecomment-1152070209

Can you add a comment explaining why we need this workaround?